.gitignore

```diff
@@ -3,3 +3,4 @@ build/
 target/
 .DS_STORE
 node_modules/
+package-lock.json
```

Comment:
My understanding is that package-lock.json should be checked in? https://docs.npmjs.com/configuring-npm/package-lock-json.html